check-cla/action.yml

```diff
@@ -3,161 +3,196 @@ name: CLA check
+        path.write_text(json.dumps(signees, indent=2))
+
+    # if unsigned, create PR
+    - name: Create PR with new CLA signee
+      uses: peter-evans/create-pull-request@v4
+      id: pull
+      if: steps.metadata.outputs.has_signed == 'false'
       with:
-        token: ${{ inputs.infrastructure_token }}
-        branch: cla-${{ github.actor }}
-        commit-message: Adding CLA singee ${{ github.actor }}
-        title: Adding CLA singee ${{ github.actor }}
+        push-to-fork: ${{ inputs.cla_fork }}
+        token: ${{ inputs.cla_token }}
+        branch: cla-${{ steps.metadata.outputs.contributor }}
+        delete-branch: true
+        commit-message: Adding CLA singee ${{ steps.metadata.outputs.contributor }}
+        author: ${{ inputs.cla_author }}
+        committer: ${{ inputs.cla_author }}
+        title: Adding CLA singee ${{ steps.metadata.outputs.contributor }}
         body: |
-          Adding CLA signee @${{ github.actor }}
+          Adding CLA signee @${{ steps.metadata.outputs.contributor }}
 
-          Xref ${{ github.event.pull_request.url }}
+          Xref ${{ steps.metadata.outputs.url }}
 
-    # create sticky comment if not signed
-    - name: Create comment
+    # if unsigned, create sticky comment
+    - name: Create comment regarding missing CLA signature
       uses: marocchino/sticky-pull-request-comment@v2
-      if: steps.contributors.outputs.hasSigned == 'false'
+      if: steps.metadata.outputs.has_signed == 'false'
       with:
-        number: context.issue.number
-        message: |
-          We require contributors to sign our [Contributor License Agreement](https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement) and we don't have one on file for @${{ github.event.pull_request.user.login }}.
+        number: ${{ steps.metadata.outputs.number }}
+        # GitHub flavored markdown reinvents how paragraphs work, adjoined lines of text are not
+        # concatenated so instead we rely on YAML multi-line + extra newlines
+        message: >-
+          [cla]: https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement
+
+
+          We require contributors to sign our [Contributor License Agreement][cla] and we don't
+          have one on file for @${{ steps.metadata.outputs.contributor }}.
+
 
-          In order for us to review and merge your code, please e-sign the [Contributor License Agreement PDF](https://conda.io/en/latest/contributing.html#conda-contributor-license-agreement). We then need to manually verify your signature, merge the PR (${{ steps.cla-pr.outputs.pull-request-url }}), and ping the bot to refresh the PR.
+          In order for us to review and merge your code, please e-sign the
+          [Contributor License Agreement PDF][cla]. We then need to manually verify your
+          signature, merge the PR (${{ steps.pull.outputs.pull-request-url }}), and ping the bot
+          to refresh the PR.
```

Comment:
@kathatherine this is the updated missing CLA message, is this good in light of https://github.com/conda/conda-docs/issues/854

To see an example of this in action see https://github.com/conda-sandbox/test-check-cla/pull/45

Reply:
Hopefully the PR link will help with anyone who just reads to the CLA signing link and stops (like what happened with some of the PyCon sprinters I was working with). Maybe two links will encourage finishing the paragraph. :P This works for me.